boot.php

```diff
@@ -1,4 +1,20 @@
 <?php
+
+// INDEX PREVENT in SEARCH_IT
+if (rex_addon::get('yform')->isAvailable()) {
+    rex_extension::register('SEARCH_IT_INDEXARTICLE', function (rex_extension_point $ep) {
+        $article = $ep->getParam('article');
+        $cat = $article->getCategory();
+        if ($article->getValue('status') == 2) {
```

Comment:
Die `2` kann bei anderen Instanzen etwas ganz anderes bedeuten als der Status von accessdenied.

Reply:
Das ist wohl möglich. Aber in accessdenied ist es nun mal der Status der verwendet wird. Ich bin für jeden Vorschlag offen es eindeutiger zu machen.

Reply:
👍🏻